org.keshavnrj.wonderwall.yaml

```diff
@@ -0,0 +1,46 @@
+app-id: org.keshavnrj.wonderwall
```

Comment:
This is supposed to match a domain that exist our you are in control (or a namespace if it's a hosting like github/gitlab/sourceforge).

Reply:
Can i put my website's domain name here? com.ktechpit.wonderwall would be fine?
If this is optional i would prefer the keep the previous values since i will require to change it in many places. :(

Reply:
it is not optional. app-id are meant to be unique, so if you don't own the namespace, it's just open for trouble.